__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -1,5 +1,17 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`toMatchImageSnapshot dumpDiffToConsole imgSrcString is added to console message when dumpDiffToConsole is true 2`] = `
+"Expected image to match or be a close match to snapshot but was 0% different from snapshot (0 differing pixels).
```

Comment:
The failure with a 0% diff is kind of confusing. Can you change the test to a more realistic example?

Reply:
updated with more realistic settings